api.swagger.yaml

```diff
@@ -1141,7 +2034,7 @@ components:
           type: string
           example: I'm taking annual leave next Thursday. Cover to be announced shortly
         timestamp:
-          description: This is timestamp (in seconds) of when the feed message was created
+          description: This is timestamp of when the feed message was created
```

Comment:
Did you mean to change this or was it a copy/paste error?

Reply:
updated